features/bevragen/persoon/ouder/dev/autorisatie-gba.feature

```diff
@@ -9,6 +9,8 @@ Functionaliteit: autorisatie oudergegevens Persoon
       | Lisette           |
 
   Rule: Vragen met fields gevraagd wordt om ouders.ouderAanduiding, wanneer de afnemer niet geautoriseerd is voor ten minste één gegeven van ouder 1 én ten minste één gegeven van ouder 2, geeft een foutmelding
```

Comment:
Wordt hier het volgende gezegd?
Voor het vragen van ouderaanduiding met fields, moet de afnemer geautoriseerd zijn voor ten minste één gegeven van ouder 1 én ten minste één gegeven van ouder 2 uit de groepen 01, 02, 03, 04 en 62. De impliciete autorisatie voor het ontvangen van inOnderzoek op oudergegevens wordt voor het ouderAanduiding veld niet meegenomen.

Vraag: geldt deze rule alleen voor het ouderAanduiding veld of geldt het ook voor de andere ouder velden dat de autorisatie voor het ontvangen van inOnderzoek op oudergegevens niet wordt meegenomen?

Reply:
ik neem de tekst die je voorstelt over voor de rule:
```
Rule: Voor het vragen van ouderaanduiding met fields, moet de afnemer geautoriseerd zijn voor ten minste één gegeven van ouder 1 én ten minste één gegeven van ouder 2 uit de groepen 01, 02, 03, 04 en/of 62.
    De impliciete autorisatie voor het ontvangen van inOnderzoek op oudergegevens wordt voor het ouderAanduiding veld niet meegenomen.
```

> Vraag: geldt deze rule alleen voor het ouderAanduiding veld of geldt het ook voor de andere ouder velden dat de autorisatie voor het ontvangen van inOnderzoek op oudergegevens niet wordt meegenomen?

Alle andere velden die je mag vragen met fields hebben een eigen autorisatie.

Deze rule geldt wel voor type van nationaliteiten en verblijfplaats, dan uiteraard met andere groepen. Voor nationaliteit is er nog geen autorisatie feature. In de verblijfplaats autorisatie feature is deze regel nog niet opgenomen.
Ik zal daarvoor een aparte PR maken